fix: Changed documentation version process (#650)
codacy/chart: RELEASE.md

    git checkout -b 'release-x.x.x'
    ```

-   [ ] 4.  Tag the commit with a release candidate version following the pattern `x.x.x-RC-0`

    For example:

    ```bash
    git tag 'x.x.x-RC-1'
    ```

    This is a requirement so that we can fill in values for `engine` and `documentation` versions on the next step.

-   [ ] 5.  Update Dependencies

    Let's assume that `requirements.yaml` file should have the correct dynamic versions configured.

    Run the following command:

    ```bash

>     make create_version_file update_dependencies

Author: Here we create the version file first and then are free to update dependencies and versions on the values files.

    ```

>     First, a `.version` file will be created based off of the most recently created tag, which is the one you have created on the previous step.
> 
>     Then, the command will update the `requirements.lock` with the latest versions and freeze the `global.workers.config.imageVersion` version on `./codacy/values.yaml`.

Reviewer: Edit to simplify the extra context about what the Makefile targets do:

```suggestion
    These Makefile targets:

    -   Create a `.version` file based on the most recent tag that you created on the previous step.
    -   Update the `requirements.lock` with the latest versions and freeze the `global.workers.config.imageVersion` version on `./codacy/values.yaml`.
```
